Gulpfile.js

```diff
@@ -21,6 +21,7 @@ var paths = {
             'node_modules/jquery/dist/jquery.min.js',
             'node_modules/semantic-ui-css/semantic.min.js',
             'src/Sylius/Bundle/UiBundle/Resources/private/js/**',
+            'src/Sylius/Bundle/UserBundle/Resources/public/js/sylius-user.js'
```

Comment:
I'm not sure whether we should share assets between old and new administration panel. IMHO even if the code stays the same, copying it to a new, private location will be a better option (so we could delete the pubic ones since we're finished with old & rusty backend)